kedro/framework/cli/jupyter.py

```diff
@@ -21,19 +21,13 @@
 from kedro.framework.project import validate_settings
 from kedro.framework.startup import ProjectMetadata
 
-CONVERT_ALL_HELP = """Extract the nodes from all notebooks in the Kedro project directory,
-including sub-folders."""
-
-OVERWRITE_HELP = """If Python file already exists for the equivalent notebook,
-overwrite its contents."""
-
 
 class JupyterCommandGroup(click.Group):
     """A custom class for ordering the `kedro jupyter` command groups"""
 
     def list_commands(self, ctx: click.Context) -> list[str]:
         """List commands according to a custom order"""
-        return ["setup", "notebook", "lab", "convert"]
+        return ["setup", "notebook", "lab"]
```

Comment:
Wondering if this custom class is needed at all? 🤔

Reply:
If you remove it does it maintains the same orders?

As the docstring mention, the class is created for a custom order vs alphabetical. The purpose of this is moving the deprecated function to the bottom.

Reply:
If you remove it, it will be returned alphabetically (lab, notebook, setup). But since the deprecated "convert" command is now gone, maybe that is okay?

Reply:
I'm also working on https://github.com/kedro-org/kedro/pull/3961 and by removing `tools/cli.py` this specific `list_commands()` method actually becomes uncovered by tests. I've been struggling to write a test for it, so would be perfect if we decide it's not needed altogether 😄

Reply:
Yeah, we've removed `convert` i think it makes sense to just get rid of it!

Reply:
Done!